mappings/net/minecraft/client/render/debug/GoalSelectorDebugRenderer.mapping

```diff
@@ -1,7 +1,16 @@
 CLASS net/minecraft/class_4205 net/minecraft/client/render/debug/GoalSelectorDebugRenderer
 	CLASS class_4206 GoalSelector
 		FIELD field_18782 pos Lnet/minecraft/class_2338;
-		FIELD field_18784 name Ljava/lang/String;
+		FIELD field_18784 label Ljava/lang/String;
```

Comment:
This fields should be unmapped. We don't know what's put in it, and it's impossible to know if it's a name (the ID of the goal selector), a label for debugging, or something else.

Reply:
It's the only string in the class, and the first parameter to `drawString`.

```
DebugRenderer.drawString(goalSelector5.label, double6, double8, double10, integer12);
```

Without knowing what it contains, what else can it be but a label?

Reply:
> Without knowing what it contains

That's exactly the problem. Knowing what it contains, it can be one of many specific things. You shouldn't name the field `label` if you don't know that it doesn't contain something more specific.

It would be the same as naming an unused field `something`.

Reply:
It's being used like a label though.

Reply:
Also the existing name that was there when I started was `name` so I'm inclined to keep to the semantics.

Reply:
Also consider that I'm not naming these in isolation. The bee and villager classes are very similar with the same fields. Based on the usages in one, you can make inferences that the other is probably the same.